### DISCUSSION TITLE
configuration problems

### DISCUSSION THREAD

**Discussion by francy202111111** [iisgiua/giuaschool#653] [Richieste di aiuto] [answered]
Hi, I can't configure the Giua School register for my school

**Comment by trinko** [comment, marked as answer]
This application is intended for Italian schools: I don't think it is useful elsewhere.
Anyway, you should:
- first of all, [read the docs](https://iisgiua.github.io/giuaschool-docs/configure.html);
- ask a specific question about exactly what you can't do.

**Reply by francy202111111** [reply]
quando clicco il pussante continua non mi fa andare avanti

**Reply by trinko** [reply]
Quindi stai parlando dell'installazione, non della configrazione.
A quale passo si blocca? Che messaggio di errore viene mostrato? Hai seguito le indicazioni sulla installazione riportate nella documentazione? 

Se non mi dai informazioni precise non posso aiutarti.

**Reply by francy202111111** [reply]
![giu](https://github.com/iisgiua/giuaschool/assets/129275539/4fb6f6d1-003b-4a2a-b6ec-ab3c48661c74)

quando clicco continua  non succede niente

**Comment by trinko** [comment]
Scusa, ma se non spieghi cosa stai facendo come faccio a capire il problema? A che pagina si riferisce l'immagine che hai caricato? Stai installando? O che altro stai facendo?

Se non spieghi bene il problema, è impossibile aiutarti.

**Reply by francy202111111** [reply]
buongiorno durante installazione del registro ho messo tutti i dati necesari  e clicco  il tasto continua a fine pagina ma cliccando il tasto continua non succede niente sto parlando della  pagina istallazione

**Comment by trinko** [comment]
Per prima cosa controlla la versione di PHP che stai usando: è richiesta una **versione >= 7.4**

Poi, sei sicuro di aver scaricato il codice giusto? Il file di installazione lo trovi sempre dal link **"scarica ultima versione"**: [vedi qui](https://iisgiua.github.io/giuaschool-docs/latest-release.html)

**Comment by francy202111111** [comment]
niente non va lo stesso 
me lo può configurare  lei _perfavore_ ?

**Comment by trinko** [comment]
No, mi spiace, non svolgo questo servizio.

Se vuoi solo provare l'applicazione, ti consiglio invece di utilizzare la demo online: [vedi qui](https://github.com/iisgiua/giuaschool/discussions/586)

Se invece vuoi risolvere il problema, mi devi dare più informazioni:
- quale servizio di hosting stai usando?
- quale versione PHP stai usando?
- quale versione del database MySql stai usando?

**Reply by francy202111111** [reply]
sono riuscito
![bp](https://github.com/iisgiua/giuaschool/assets/129275539/df2dabff-ad32-4d54-bd31-4cb3410d1973)
 ad cliccare continua mi da questo

**Reply by francy202111111** [reply]
il mio hoste e register.it
la versione e 11.00

**Comment by trinko** [comment]
Scusa, ma se vuoi usarlo sul tuo computer devi prima attivare il server Apache con il PHP...
Non puoi semplicemente aprire il file dell'installazione con il browser.

Se vuoi usare giua@school sul tuo PC, consiglio di utilizzare le **immagini docker**: segui le indicazioni della [documentazione](https://iisgiua.github.io/giuaschool-docs/install-docker.html)

**Comment by francy202111111** [comment]
ecco le info
> * quale servizio di hosting stai utilizzando? register.it
> * quale versione PHP stai utilizzando? 
> * quale versione del database MySql stai utilizzando? 11.00
